src/parser.js

```diff
@@ -32,8 +32,13 @@ const INIT_MASK = 1;
 const GETTER_MASK = 2;
 const SETTER_MASK = 4;
 
-const STRICT_MODE_RESERVED_WORD = [
-  "implements", "interface", "package", "private", "protected", "public", "static", "yield", "let"];
+// Empty parameter list for ArrowExpression
+const ARROW_EXPRESSION_PARAMS = "CoverParenthesizedExpressionAndArrowParameterList";
+
+const STRICT_MODE_RESERVED_WORD = {
+  "implements": true, "interface": true, "package": true, "private": true, "protected": true,
+  "public": true, "static": true, "yield": true, "let": true
```

Comment:
I opt to assign a falsey value to these kinds of maps in case someone is tempted to check existence via reguar member access. Also, we can use `__proto__: null` if we want to use an `in` test.